docs/guide/gui/image.md

```diff
@@ -1,56 +1,56 @@
 # UIImage
 
-[UIImage](/api/classes/UIImage) 组件提供图片的显示功能, 跟字体组件一样，图片组件依赖于 `Atlas` 图集，你需要先加载精灵图集到引擎中，然后指定精灵赋予图片组件供渲染。
+[UIImage](/api/classes/UIImage) component provides image display functionality. Like the font component, the image component relies on the `Atlas` atlas. You need to first load the sprite atlas into the engine, and then assign the sprite to the image component for rendering.
 
-> 默认 `UIImage` 组件未设置精灵，将以白色方块形式呈现
+> The default `UIImage` component has no sprite set and will be presented in white squares
 
 ```ts
 import { Engine3D } from '@orillusion/core';
 
-// 创建用于显示UI的面板
+// Create a panel for displaying the UI
```

Comment:
create---adding---create---loading---...
perhaps to  use the same format, verb or verb-ing